### PR TITLE
Feat: #1051 - Update Active/Inactive Button copy to Configure/Not Configured

### PR DESCRIPTION
**Pull request by ajay-sentry**
# Description

Updates "Inactive" and "Active" button Copy on the main code coverage page to "Configured" and "Not Configured"

Related to [this ticket](https://github.com/orgs/codecov/projects/31/views/12?pane=issue&itemId=50147857&sortedBy%5Bdirection%5D=asc&sortedBy%5BcolumnId%5D=Status)

# Notable Changes

Nothing really, mostly copy updates. One "new" pattern I didn't see used much is using the "exact: true" keyword on tests which is kinda handy when you have things with similar names on a page

# Screenshots

Screenshot 2024-02-20 at 3 27 06 PM

https://github.com/codecov/gazebo/assets/159853603/c12f58c5-2931-4e2c-ba25-91ad249a5019

# Link to Sample Entry

# Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. In 2022 this entity acquired Codecov and as result Sentry is going to need some rights from me in order to utilize my contributions in this PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

**Review by RulaKhaled — COMMENTED** [review]
looking at https://deploy-preview-2610--gazebo-staging.netlify.app/gh/codecov?repoDisplay=Not%20Configured we are viewing configured repos in the not configured list, it looks like we're not sending `is_activated: false` to the repos list query, do you mind looking into this?

**Comment by ajay-sentry** [comment]
@RulaKhaled good catch! I didn't even notice the names were the same. I think the bug is here: https://github.com/codecov/gazebo/blob/a68c47dc38530c70844d55ffc407b780e0c9d28d/src/shared/ListRepo/ReposTableTeam/ReposTableTeam.tsx#L149-L152 due to the space in the text now. Going to try to figure out a clean solution

**Comment by ajay-sentry** [comment]
@RulaKhaled Not ideal, but I ended up reverting to the old way of doing the lookup on the const, but adding a replace for the key with a space to match the key. So "Not Configured" -> "NOT_CONFIGURED" and everything works out

File: src/shared/ListRepo/ReposTableTeam/ReposTableTeam.tsx [hunk 1]
```diff
@@ -146,9 +146,12 @@ const ReposTableTeam = ({ searchValue }: ReposTableTeamProps) => {
   const { owner } = useParams<{ owner: string }>()
 
   const repoDisplay = useContext(ActiveContext)
+
   const activated =
     repoDisplayOptions[
-      repoDisplay.toUpperCase() as keyof typeof repoDisplayOptions
+      repoDisplay
+        .replace(/\s/g, '_')
```
Comment (RulaKhaled) [inline src/shared/ListRepo/ReposTableTeam/ReposTableTeam.tsx:153]: magic

**Review by RulaKhaled — APPROVED** [review]
LGTM

File: src/shared/ListRepo/ReposTableTeam/ReposTableTeam.tsx (hunk 1 above)
Comment (ajay-sentry) [reply src/shared/ListRepo/ReposTableTeam/ReposTableTeam.tsx:153]: 😅 Gotta figure out something a little cuter next time

**Pull request merged**